File: Python version/specializations/dlv/dlv_answer_sets.py
from languages.asp.answer_sets import AnswerSets
from parsers.asp.asp_solvers_parser import ASPSolversParser


class DLVAnswerSets(AnswerSets):
    """Represents an AnswerSet for DLV."""

    def __init__(self, out, err=None):
        super(DLVAnswerSets, self).__init__(out, err)

    def _parse(self):
        ASPSolversParser.parse_dlv(self, self._output, True)

File: CSharp version/it/unical/mat/embasp/specializations/dlv/DLVAnswerSets.cs
using it.unical.mat.parsers.asp;

namespace it.unical.mat.embasp.specializations.dlv
{
    using AnswerSets = it.unical.mat.embasp.languages.asp.AnswerSets;

    public class DLVAnswerSets : AnswerSets
    {
        public DLVAnswerSets(string answerSets) : base(answerSets) { }

        public DLVAnswerSets(string @out, string err) : base(@out, err) { }

        protected internal override void Parse()
        {
            ASPSolversParser.ParseDLV(this, output, true);
        }
    }
}